selfservice/hook/verification.go

```diff
@@ -72,6 +74,12 @@ func (e *Verifier) do(r *http.Request, i *identity.Identity, f flow.Flow) error
 		if err := e.r.LinkSender().SendVerificationTokenTo(r.Context(), verificationFlow, address, token); err != nil {
```

Comment:
I think we should update the status in this method instead!

Reply:
> I think we should update the status in this method instead!

done.